omero/sysadmins/server-upgrade.txt

```diff
@@ -263,15 +265,16 @@ See :ref:`jvm_memory_settings` for more information.
 OMERO.web server configuration
 """"""""""""""""""""""""""""""
 
-The generated web-server configurations for Nginx and Apache have been
-revised in OMERO 5.1. It is highly recommended that you regenerate your
-:ref:`wsgi_configuration`, remembering to merge in any of your own
-modifications if necessary.
-See :doc:`/sysadmins/whatsnew` for details of changes.
+The FastCGI support was removed in OMERO 5.2 and OMERO.web can be deployed
+using WSGI :ref:`unix_omero_web_deployment`. If you already deployed
+OMERO.web using WSGI it is highly recommended that you regenerate your
+:ref:`unix_omero_web_deployment`, remembering to merge in any of your own
+modifications if necessary. See :doc:`/sysadmins/whatsnew` for details
+of changes.
```

Comment:
Also you've linked to `:ref:unix_omero_web_deployment` twice in the same paragraph, once should be enough.